packages/synapse-react-client/src/components/QuerySortSelector.tsx

```diff
@@ -3,18 +3,30 @@ import { SortConfiguration } from './CardContainerLogic'
 import { useQueryContext } from './QueryContext/QueryContext'
 import { SortDirection, SortItem } from '@sage-bionetworks/synapse-types'
 import { Typography } from '@mui/material'
-import Select from 'react-select'
-import {
-  EnumOption,
-  findValueOption,
-  Control,
-} from './entity/annotations/CustomSelectWidget'
+import Select, { components, ControlProps, GroupBase } from 'react-select'
+import { findValueOption } from './SchemaDrivenAnnotationEditor/widget/SelectWidget'
 import { useQueryVisualizationContext } from './QueryVisualizationWrapper'
 
 export type QuerySortSelectorProps = {
   sortConfig: SortConfiguration
 }
 
+export type EnumOption = { value: string; label: string }
+
+/**
+ * We want to apply the 'form-control' bootstrap class to react-select's Control component, and the easiest way to do that is to make a custom version
+ */
+export const Control = ({
+  children,
+  ...rest
+}: ControlProps<any, boolean, GroupBase<any>>) => {
+  return (
+    <components.Control {...rest} className="form-control">
+      {children}
+    </components.Control>
+  )
+}
```

Comment:
Moved from `CustomSelectWidget`, since these are now only used in this file.

Reply:
We still want to use the bootstrap form-control css class here?

Reply:
Not sure. Left out of scope for this PR, since this isn't part of the annotations editor, it just borrowed a component from the previous implementation.